bus/client.go

```diff
@@ -67,6 +67,44 @@ func (c *Client) WalletOutputs() (resp []wallet.SiacoinElement, err error) {
 	return
 }
 
+// estimatedSiacoinTxnSize estimates the txn size of a siacoin txn without file
+// contract given its number of outputs.
+func estimatedSiacoinTxnSize(nOutputs uint64) uint64 {
+	return 1000 + 60*nOutputs
+}
+
+// SendSiacoins is a helper method that sends siacoins to the given outputs.
+func (c *Client) SendSiacoins(scos []types.SiacoinOutput) (err error) {
```

Comment:
could've been defined on the `TestCluster`?

Reply:
Agreed -- generally speaking, we want `Client` methods to be simple. It should be easy to implement a client in any language. I'm ok adding a `/wallet/send` endpoint; even though it's *technically* redundant, it's a very common operation. We don't want the UI to be manually building txns.